src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -874,7 +874,9 @@ namespace OpenLoco::Vehicles
     bool VehicleHead::updateLand()
     {
         Vehicle2* vehType2 = _vehicleUpdate_2;
-        if ((!(vehType2->var_73 & Flags73::isBrokenDown) || (vehType2->var_73 & Flags73::isStillPowered)) && status == Status::approaching)
+
+        // If dont have any running issue and is aproaching
+        if (vehType2->hasNo73Flags() && (status == Status::approaching))
```

Comment:
I think this might be wrong actually, If you read it like this:
```cpp
if (
    (!has73Flags(Flags73::isBrokenDown) || has73Flags(Flags73::isStillPowered)) 
    && status == Status::approaching
)
```
there is a difference, the first condition checks if isBrokenDown is not set OR has the flag isStillPowered, that condition would be false for when no flags are set if I'm not mistaken